content/02.main-text.md

```diff
@@ -257,3 +263,25 @@ Manubot provides an ideal platform for perpetual reviews [@arxiv:1502.01329; @ta
 We invite the scientific community to adapt and build upon our experience and open software.
+
+### The open science movement
+
+Science is undergoing a transition towards openness.
+The internet provides a global information commons, where scholarship can be publicly shared at a minimal cost.
+For example, open access publishing provides an economic model that encourages maximal dissemination and reuse of scholarly articles [@doi:10.12688/f1000research.8460.3].
+Pressure from funders, libraries, self-archiving, and piracy are set to cement open access as the primary model for publishing the scientific record [@doi:10.7717/peerj.4375; @doi:10.7554/eLife.16800; @doi:10.7554/eLife.32822; @doi:10.1038/d41586-018-05754-1].
+However, open science extends beyond just open access to journal articles.
+Open licensing solves legal barriers to content reuse, enabling any type of scholarly output to become part of the commons [@doi:10.1109/MCSE.2009.19; @doi:10.1038/536016a].
+
+The push for increased reproducibility and the opportunity to reuse data and code for new investigations has begot a movement to make all research outputs public, unless there are bonafide privacy or security concerns [@doi:10.1126/science.aah6168; @doi:10.1038/nature10836; @doi:10.1371/journal.pbio.1001195].
+New tools and services make it increasingly feasible to publicly share the unabridged methods of a study, especially for computational research, which consists solely of software and data.
+Meanwhile, open source software has enabled computational infrastructure to reach new heights, in terms of quantity, quality, and recruitment of contributions.
+Greater openness in both research methods and publishing is creating an opportunity to redefine peer review and the role journals play in communicating science [@doi:10.12688/f1000research.12037.1].
+At the extreme is real-time open science, whereby studies are performed entirely in the open from their inception [@doi:10.1242/dmm.003285].
+Many such research projects have now been completed, benefiting from the associated early-stage peer review, additional opportunity for online collaboration, and increased visibility [@doi:10.3897/rio.1.e7547; @doi:10.7554/eLife.26726].
+
+For open science to accelerate the pace of discovery and encourage maximal contribution to the public scientific record, researchers will require a comprehensive suite of tools that replace closed & proprietary services with open and community-driven services.
+We developed Manubot with the goal of adapting the the open source software workflow for scholarly writing.
+As such, using Manubot requires some technical expertise, but the benefits are manyfold.
+Specifically, Manubot demonstrates a system for publishing that is transparent & reproducible, immediate & permissionless, versioned & automated, collaborative & open, linked & provenanced, decentralized & hackable, interactive & annotated, and free of charge.
```

Comment:
I like the adjectives but the punctuation is throwing me off.  Could this be a standard list?

This alone would actually satisfy me for #46.  We could say more about the Manubot roadmap, but this stands alone very well.

Reply:
For some reason I liked the flow of the coupled adjectives, but we could also do:

> Manubot demonstrates a system for publishing that is transparent, reproducible, immediate, permissionless, versioned, automated, collaborative, open, linked, provenanced, decentralized, hackable, interactive, annotated, and free of charge

@agitter do you like that more?

Reply:
@dhimmel your version was more stylish, but I do prefer the bland version for a formal article.